lib/ex_unit/lib/ex_unit.ex

```diff
@@ -457,6 +461,10 @@ defmodule ExUnit do
     end
   end
 
+  defp put_exit_status(opts) do
+    Keyword.put_new(opts, :exit_status, 2)
+  end
```

Comment:
This function is not necessary. Instead, put the default value in lib/ex_unit/mix.exs. The default values in this file are only the ones that need to be computed dynamically. :)